client/src/main/scala/org/apache/celeborn/client/LifecycleManager.scala

```diff
@@ -616,7 +611,7 @@ class LifecycleManager(val appUniqueId: String, val conf: CelebornConf) extends
     reply(mapperAttemptFinishedSuccess)
   }
 
-  def handleUnregisterShuffle(
+  def unregisterShuffle(
       shuffleId: Int): Unit = {
     if (getPartitionType(shuffleId) == PartitionType.REDUCE) {
```

Comment:
@RexXiong curiously, is unregister shuffle required for Flink? or it's not related to engine, just related to MAP/REDUCE partition types